std_execution.bs

```diff
@@ -2950,79 +2944,90 @@ namespace std::execution {
     template &lt;class CPO>
     struct get_completion_scheduler_t;
 
+    struct schedule_t;
+    struct transfer_just_t;
+    struct on_t;
+    struct transfer_t;
+    struct schedule_from_t;
+    struct then_t;
+    struct upon_error_t;
+    struct upon_done_t;
+    struct let_value_t;
+    struct let_error_t;
+    struct let_done_t;
+    struct bulk_t;
+    struct split_t;
+    struct when_all_t;
+    struct when_all_with_variant_t;
+    struct transfer_when_all_t;
+    struct transfer_when_all_with_variant_t;
+    struct done_as_optional_t;
+    struct done_as_error_t;
+    struct ensure_started_t;
+    struct start_detached_t;
```

Comment:
It has always seemed odd to me that all these CPOs were dumped into the same namespace. Let's at least move the adaptors and consumers (everything from `transfer_just_t`) into its own namespace.